resources/assets/components/CampaignsTable.js

```diff
@@ -110,9 +117,24 @@ const CampaignsTable = ({ isOpen, filter }) => {
       <table className="table">
         <thead>
           <tr>
-            <td>Campaign</td>
-            <td>Pending</td>
-            <td>Accepted</td>
+            <OrderableHeading
```

Comment:
"Orderable" is a hard word to say

Reply:
And to spell! 😂 Any ideas on a better term for this? Maybe `SortableHeading`?

Reply:
SORTABLE I like that better!

Reply:
Yeah me too! Swapped in 49f6f51.